sonar-core/src/main/resources/org/sonar/l10n/core.properties

```diff
@@ -1302,6 +1302,11 @@ coding_rules.remediation_function.coeff=Coeff
 coding_rules.remediation_function.offset=Offset
 coding_rules.remediation_function.constant=Constant
 
+coding_rules.scope.title=Analysis Scope
+coding_rules.scope.MAIN=Main sources
+coding_rules.scope.TESTS=Test sources
```

Comment:
Typo in scope TESTS, it should be TEST => 

`coding_rules.scope.TEST=Test sources`

Reply:
(The UI is badly displaying label coding_rules.scope.TESTS when a rule has a TEST scope)

Reply:
Good catch! Thanks!